src/components/shared/SoldByCard.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+type SoldByProps = {
+  seller?: string;
+  feedback?: number;
+  sellerItems: URL | string;
+  contactSeller: URL | string;
+};
+
+function SoldByCard({
+  seller,
+  feedback,
+  sellerItems,
+  contactSeller,
+}: SoldByProps) {
+  return (
+    <div className="bg-white rounded-lg w-52 h-52 mt-2 drop-shadow-lg">
+      <h3 className="text-dark text-lg font-medium ml-4 pt-3">Sold by:</h3>
+      <p className="text-primary font-medium ml-4 mt-2">{seller}</p>
```

Comment:
I think this should also be a link in case a buyer wants to view all products sold by a specific person.

Reply:
Oh. I was misled then. I saw the primary colour for the text (in the ui), and instantly assumed that it was a clickable link.